### PR TITLE
Add time and remove expenses warning on pension form

### PR DESCRIPTION
**Pull request by jbalboni**
These are the last two changes I had in my notes.

screen shot 2017-08-04 at 12 44 40 pm
screen shot 2017-08-03 at 9 45 14 am

### CODE REVIEW

File: src/js/pensions/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ class ConfirmationPage extends React.Component {
           <ul className="claim-list">
             <li>
               <strong>Date received</strong><br/>
-              <span>{moment(form.submission.submittedAt).format('MMM D, YYYY')}</span>
+              <span>{moment(form.submission.submittedAt).utcOffset('-0600').format('MMM D, YYYY h:mm A [CT]')}</span>
```
Comment (jbalboni) [inline src/js/pensions/containers/ConfirmationPage.jsx]: Not sure this handles DST correctly.

File: src/js/pensions/config/form.js [hunk 2]
```diff
@@ -1385,13 +1374,7 @@ const formConfig = {
               'ui:options': {
                 expandUnder: 'view:spouseHasOtherExpenses'
               }
-            }),
-            'view:spouseOtherExpensesWarning': {
```
Comment (cehsu) [inline src/js/pensions/config/form.js]: @jbalboni should we also remove the accompanying schema property for this (lines 1372-1375)?

**Review by cehsu — APPROVED** [review]
Looks good to me. Had a potential zombie code question.

File: src/js/pensions/config/form.js (hunk 2 above)
Comment (jbalboni) [inline src/js/pensions/config/form.js]: Yes! Missed that one.

**Pull request merged**